test/logging/test_tensorboard.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+
+import tarfile
+import glob
+
+import netket as nk
+from jax.nn.initializers import normal
+from jax import numpy as jnp
+
+from .. import common
+
+pytestmark = common.skipif_mpi
+
+
+@pytest.fixture()
+def vstate(request):
+    N = 8
+    hi = nk.hilbert.Spin(1 / 2, N)
+
+    ma = nk.models.RBM(
+        alpha=1,
+        dtype=float,
+        hidden_bias_init=normal(),
+        visible_bias_init=normal(),
+    )
+
+    return nk.vqs.MCState(
+        nk.sampler.MetropolisLocal(hi),
+        ma,
+    )
+
+
+def test_tblog(vstate, tmp_path):
+    path = str(tmp_path) + "/dir1/dir2"
+
+    log = nk.logging.TensorBoardLog(path)
+
+    for i in range(10):
+        log(i, {"Energy": jnp.array(1.0), "complex": jnp.array(1.0 + 1j)}, vstate)
+
+    log.flush()
+    del log
+
+    files = glob.glob(path + "/*")
+    assert len(files) >= 1
+
+
+def test_lazy_init(tmp_path):
+    # skip test if qutip not installed
+    pytest.importorskip("tensorboardX")
```

Comment:
"qutip"? Also, shouldn't this skip apply to the other test too then?

Reply:
just a typo. i also added it to the other test